webhook_testing_gateway.go

```diff
@@ -0,0 +1,34 @@
+package braintree
+
+import (
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+// WebhookTestingGateway exports actions only available in the sandbox environment.
+type WebhookTestingGateway struct {
+	*Braintree
+	apiKey apiKey
+}
+
+// Request simulates an incoming webhook notification request
+func (g *WebhookTestingGateway) Request(method, endpoint, kind, id string) (*http.Request, error) {
```

Comment:
I think we can drop the `method` and `endpoint` arguments here. `method` will always be `POST` and `endpoint` _should_ be irrelevant and the developer could set it manually on the `http.Request` they get back.